query/deployment.yaml

```diff
@@ -83,3 +132,32 @@ spec:
          secret:
            optional: false
            secretName: ssl-config-query
+{% if not deploy %}
```

Comment:
I'm fine if you just make this unconditional.  Nobody is going to be using it seriously and I feel like it is going to be changing quickly.